config/recurring.yml

```diff
@@ -0,0 +1,10 @@
+# production:
```

Comment:
@shipstar we didn't use to have any leads on running the recurring functionality teased in our codebase before. This came along for the ride with `bin/rails solid_queue:install`. Should we keep it or remove it?